typescript-react/src/components/Form/Form.module.css

```diff
@@ -21,12 +21,20 @@
     padding: 10px;
     box-shadow: -1px -1px 3px 0 btnPrimaryColor;
     background-color: loginBodyColor;
-    z-index: 5;
+    z-index: 110;
   }
 
   .FormComponent {
+    margin-bottom: 72px;
```

Comment:
Hm, why? I don't have the app in front of me, but this looks like it will leave quite a bit of whitespace on desktop screens :thinking: Marko will complain about whitespace creep again!

Reply:
I merged this part into this line and removed it below.

@media (max-width: 980px) {	
  .FormComponent {	
    margin-bottom: 72px;	
  }	
}

Reply:
Ah, my bad, needed to expand above to see the media scope :+1: